torch/csrc/jit/ir.cpp

```diff
@@ -250,6 +250,175 @@ std::ostream& operator<<(std::ostream & out, const Graph & g) {
   return out;
 }
 
+std::string getBaseName(std::string str) {
+  return str.substr(0, str.find_first_of('.'));
+}
+
+std::ostream& prettyPrintValue(std::ostream & out, const Value* val) {
+  auto scope = val->node()->scope();
+  auto unique = val->uniqueName();
+  if (val->node()->kind() == prim::Constant) {
+    if (scope->hasConstant(unique)) {
+      out << scope->getConstant(unique);
+    } else {
+      throw std::runtime_error("const not found for : " + unique);
+    }
+  } else {
+    auto name = val->readableName();
+    if (scope->hasNameAlias(unique)) {
+      name = scope->getNameAlias(unique);
+    }
+
+    if (isdigit(name.at(0))) {
+      out << "%";
+    }
+    out << name;
+  }
+return out;
+}
+
+std::ostream& prettyPrintInputs(std::ostream & out, const Node* node) {
+  out << "(";
+  auto delimiter = "";
+  for (auto in_value : node->inputs()) {
```

Comment:
Nit:  prefer `const auto&` or `const auto*`, right now it's hard to know if the iterated thing is a pointer or a value